styleguide/babel.config.js

```diff
@@ -4,9 +4,14 @@ module.exports = {
     '@babel/preset-typescript',
     '@babel/preset-react'
   ],
-  plugins: [
-    '@babel/plugin-proposal-export-namespace-from',
-    '@babel/plugin-proposal-throw-expressions',
-    '@babel/plugin-proposal-class-properties'
+  overrides: [
```

Comment:
This was the only way I could get `preset-env` options properly merged, for some reason I have to define it once within the `preset` array then again within `overrides->presets`. Could use some eyes here too @NavyAdmiral  @rupurt